src/brackets.js

```diff
@@ -113,6 +113,7 @@ define(function (require, exports, module) {
     require("utils/Resizer");
     require("LiveDevelopment/main");
     require("utils/ColorUtils");
+    require("utils/Parameters3DUtils");
```

Comment:
Because you only ever call `Parameters3DUtils` methods within your extension, I don't think it belongs in `utils/`.  Instead, you should put it in `src/extensions/default/Inline3DParametersEditor`.